google/auth/aws.py

```diff
@@ -540,7 +562,12 @@ def _get_region(self, request, url):
 
         if not self._region_url:
             raise exceptions.RefreshError("Unable to determine AWS region")
-        response = request(url=self._region_url, method="GET")
+
+        headers = None
+        if session_token is not None:
+            headers = {"X-aws-ec2-metadata-token": session_token}
```

Comment:
this can be done in one line:
```suggestion
        headers = {"X-aws-ec2-metadata-token": session_token} if session_token else None
```